### PR TITLE
Update @guardian/types themes

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Integrate the changes brought forward by https://github.com/guardian/types/releases/tag/v0.5.2, which are currently breaking the builds.

List of changes:

- Wrangle types
- This also adds a script to force NPM, as I always trip up on this one and mentally default to `yarn`.
- Make @dependabot happy: #65 and #78

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "ts-loader": "^8.0.5"
   },
   "scripts": {
+    "preinstall": "if [[ $npm_execpath =~ 'yarn' ]]; then echo 'Use NPM!' && exit 1; fi",
```
Comment (JamieB-gu) [inline package.json:31]: 😆

Comment (mxdvl) [reply package.json:31]: I feel like this is the only repo using NPM, and I always trip up!

**Review by webb04 — APPROVED** [review]
LGTM 👍

**Review by JamieB-gu — APPROVED** [review]
Things are happening so quickly! 👍 

Do you think this should be a major version bump to `2.0.0` given that this changes the API?

**Comment by mxdvl** [comment]
@JamieB-gu Yes, technically, this should be a v2.0 bump. I don’t see the fact that it coincided with the v1.0 release to be used in AR related.

Unless you consider the integration of `Theme = Pillar | Special` the reason for v1.0 and therefor this would be a fix, the wrong interface being the problem.

### Cast your vote below
🚀 = v2.0.0
🎉 = v1.0.1

**Comment by JamieB-gu** [comment]
> Unless, that is, you consider the integration of `Theme = Pillar | Special` the reason for v1.0

The move to `v1.0` was just because it was ready to use in AR. The `Theme` migration in AR happened independently.

**Comment by mxdvl** [comment]
Happy to merge, but I don’t think I have the power to publish a package.

```
npm ERR! code E402
npm ERR! 402 Payment Required - PUT https://registry.npmjs.org/@guardian%2fimage-rendering - You must sign up for private packages
```

**Comment by JamieB-gu** [comment]
> Happy to merge, but I don’t think I have the power to publish a package.

Merge it!

This isn't published to npm atm. Apps-rendering is consuming it [like this](https://github.com/guardian/apps-rendering/pull/844/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R43).

**Comment by JamieB-gu** [comment]
I've released this changes [here](https://github.com/guardian/image-rendering/releases/tag/v2.0.0).

File: package.json (hunk 1 above)
Comment (mxdvl) [reply package.json:33]: Discussing this with @sndrs, there might be issues when installing this dependencies in `yarn` projects.

**Pull request merged**